packages/office-ui-fabric-react/src/components/KeytipLayer/KeytipLayer.tsx

```diff
@@ -0,0 +1,12 @@
+import * as React from 'react';
+import {
+  styled
+} from '../../Utilities';
+import { IKeytipLayerProps } from './KeytipLayer.types';
+import { KeytipLayerBase } from './KeytipLayer.base';
+import { getStyles } from './KeytipLayer.styles';
+
+export const KeytipLayer = styled(
```

Comment:
>styled [](start = 27, length = 6)

[nit] styled<IKeytipProps, IKeytipStyleProps, IKeytipStyles> for consistency with keytipContent.tsx